modules/client/examples/03-tokenVoting-client/18-undelegate-tokens.ts

```diff
@@ -0,0 +1,49 @@
+/* MARKDOWN
+---
+title: Undelegate Tokens
+---
+
+## Undelegates yout token voting power from the delegatee address
+
+This is the same as delegating to your own address.
+*/
+
+import {
+  ContextPlugin,
+  DelegateTokensStep,
+  TokenVotingClient,
+} from "@aragon/sdk-client";
+import { context } from "../index";
+
+// Instantiate the general purpose client from the Aragon OSx SDK context.
+// Instantiate a plugin context from the Aragon OSx SDK context.
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+
+// Create a TokenVoting client.
+const tokenVotingClient: TokenVotingClient = new TokenVotingClient(
+  contextPlugin,
+);
+
+
+const steps = tokenVotingClient.methods.undelegateTokens("0x1234567890123456789012345678901234567890");
```

Comment:
I would make it a bit clearer:

```suggestion
const tokenAddress = "0x1234567890123456789012345678901234567890";
const steps = tokenVotingClient.methods.undelegateTokens(tokenAddress);
```